packages/backend-storage/src/construct.ts

```diff
@@ -122,7 +127,7 @@ export class AmplifyStorage
       },
     };
 
-    this.storeOutput(props.outputStorageStrategy);
+    this.storeOutput(props.outputStorageStrategy, props.isDefault || false);
```

Comment:
```suggestion
    this.storeOutput(props.outputStorageStrategy, props.isDefault ?? false);
```

Reply:
`isDefault` can't be `0` or `""` since it's set to be `boolean` type, so I think `||` is appropriate here. Even if it was `0` or `""`, I think we can assume it means `false`.